build/dangerjs/checkChangelog.ts

```diff
@@ -21,7 +21,7 @@ const hasAddedLinesAfterVersionInChangelog = async (danger): Promise<boolean> =>
 
 const getMalformedChangelogEntries = async (danger): Promise<string[]> => {
   // +- description @githubname ([#DDDD](https://github.com/stardust-ui/react/pull/DDDD))
-  const validEntry = /^\+- .*@\S+ \(\[#(\d+)]\(https:\/\/github\.com\/(?:stardust-ui\/react|microsoft\/fluent-ui-react)\/pull\/\1\)\)$/
+  const validEntry = /^\+- .*@\S+ \(\[#(\d+)]\(https:\/\/github\.com\/(?:stardust-ui\/react|stardust-ui\/react-old|microsoft\/fluent-ui-react)\/pull\/\1\)\)$/
```

Comment:
Was this intentional?

Reply:
oh it was there to get around a weird danger.js check but it's not needed now that i realized that my problem was over zealous rename of things all across repo

Reply:
I'll revert this.